.github/actions/test-coverage/action.yml

```diff
@@ -13,7 +13,7 @@ inputs:
   duration:
     description: 'Tests maximum duration'
     required: true
-    default: '20'
+    default: '10'
```

Comment:
More than enough, we very rarely see them or use slowest tests info from here to optimize them